connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -314,7 +314,48 @@ object CheckConnectJvmClientCompatibility {
       ProblemFilters.exclude[DirectMissingMethodProblem](
         "org.apache.spark.sql.streaming.DataStreamReader.validateJsonSchema"),
       ProblemFilters.exclude[DirectMissingMethodProblem](
-        "org.apache.spark.sql.streaming.DataStreamReader.validateXmlSchema"))
+        "org.apache.spark.sql.streaming.DataStreamReader.validateXmlSchema"),
+      ProblemFilters.exclude[MissingClassProblem](
```

Comment:
@HyukjinKwon @hvanhovell can you take a look? I'm not sure why moving files from avro to sql/core breaks Spark Connect compatibility check. The avro module is not in the client anyway.

Reply:
I think we can add `org.apache.spark.sql.avro.*` on the top

Reply:
Wait, actually seems we need to do something with `checkMiMaCompatibilityWithAvroModule`. @LuciferYang I remember you took a look at this. Mind helping this whenever you find some time please? 🙏

Reply:
> I think we can add `org.apache.spark.sql.avro.*` on the top

This is reasonable. The MIMA check for Avro in connect-client mainly focuses on the `function` package, which is not within the scope of the inspection and should be skipped.